extensions/wikia/AnalyticsEngine/js/universal_analytics.js

```diff
@@ -253,17 +257,32 @@
 		return kruxSegment;
 	}
 
-	function trackBlocking(value) {
-		if (blockingTracked) {
+	function trackBlockingSourcePoint(value) {
+		if (blockingTracked.sourcePoint) {
 			return;
 		}
-		blockingTracked = true;
+		blockingTracked.sourcePoint = true;
 		_gaWikiaPush(['set', 'dimension6', value]);
 		window.ga('ads.set', 'dimension6', value);
 		guaTrackAdEvent('ad/sourcepoint/detection', value, '', 0, true);
 		guaTrackEvent('ads-sourcepoint-detection', 'impression', value, 0, true);
 	}
 
+	function trackBlockingPageFair(isBlocked) {
```

Comment:
`trackBlockingPageFair()` and `trackBlockingSourcePoint()` seems really similar, what do you think about minimizing the repeated lines?